example.js

```
@@ -12,6 +12,97 @@ function multiply(a, b) {
 const subtract = (a, b) => {
   return a - b;
 }
+// Function 1: simple addition
+export function add(a, b) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'add'. Consider to delete it or rename it.

'add' is defined here:


(lint/suspicious/noRedeclare)